rm_vk/rm_vk/MyGroupsTableViewController/ViewController/MyGroupsTableViewController.swift

```diff
@@ -19,8 +20,10 @@ final class MyGroupsTableViewController: UITableViewController {
 
     // MARK: - Private Properties
 
-    private var groups: [Group] = []
     private var networkService: NetworkServiceProtocol = NetworkService()
+    private let realm = try? Realm()
+    private var groupsToken: NotificationToken?
```

Comment:
```suggestion
    private let realm = try? Realm()
    private var groupsToken: NotificationToken?
```
можно нам на вынос?)